### PR TITLE
Fixes Glowbulbs spawning nymphs when you destroy them.

### LINKED ISSUE #14131: Welding open a Glow Bulb creates a Nymph.
Step 1. Create Glowbulb as Diona
Step 2. Apply Weldingtool
No this is apparently not supposed to happen.

### PR DESCRIPTION
**Pull request by RisingValiant**
I mentioned it to Yonnimer, no this is not intended behaviour. 
Fixes #14131

### CODE REVIEW

**Comment by RisingValiant** [comment]
!bugfix
!review

**Comment by RisingValiant** [comment]
!review

**Comment by RisingValiant** [comment]
!bugfix

File: html/Glowbulbfix.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: Changed the code for Diona Glow Bulbs so that destroying one no longer spawns a nymph.
```
Comment (SleepyGemmy) [inline html/Glowbulbfix.yml:41]: wrap this in quotation marks.
the file should also be under `html/changelogs/`.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by Arrow768 — CHANGES_REQUESTED** [review]
Oh yeah, I oversaw the changelog.
That needs to be in html/changelogs

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Remove the changelog in the html folder.

**Pull request closed without merging**